src/udev/udev-spawn.c

```diff
@@ -279,6 +280,12 @@ int udev_event_spawn(
                 free_and_replace(argv[0], program);
         }
 
+        char *found;
+        if (find_callout_binary(argv[0], &found) >= 0) {
```

Comment:
Maybe just make pin_callout_binary return path additionally, and use fexecve_or_execve then?